src/views/LatticeView.vue

```diff
@@ -0,0 +1,52 @@
+<script setup lang="ts">
+import { computed, ref } from "vue";
+import type { Scale } from "scale-workshop-core";
+import { mmod } from "xen-dev-utils";
+import ScaleLattice from "@/components/ScaleLattice.vue";
+
+// See ji-lattice-2 branch for ideas and how to extract coordinates from the scale.
```

Comment:
These comments were inteded as guidance during development. They should not end up in the main branch. If you need help with the bootstrapping I can take over the branch once everything else has been dealt with.